helpers/url_test.go

```diff
@@ -0,0 +1,30 @@
+package helpers_test
+
+import (
+	"net/url"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"go-google-scraper-challenge/helpers"
+)
+
+var _ = Describe("URL Helpers", func() {
+	Describe("#IsActive", func() {
+		Context("given the URL and the path are the same", func() {
+			It("returns the given file content", func() {
```

Comment:
```suggestion
			It("returns true", func() {
```

Reply:
Fixed in [8717863](https://github.com/carryall/go-google-scraper-challenge/pull/113/commits/8717863dc3e1767f5dbfa8244a0e6c5954bc55b3) along with the comment below